sap.go

```diff
@@ -21,8 +21,13 @@ const (
 	CORBUFFER_PRM_ADDRESS = C.CORBUFFER_PRM_ADDRESS
 
 	// registers related to horizontal flip
-	CORACQ_PRM_FLIP = C.CORACQ_PRM_FLIP
-	CORACQ_CAP_FLIP = C.CORACQ_CAP_FLIP
+	CORACQ_PRM_FLIP                    = C.CORACQ_PRM_FLIP
+	CORACQ_CAP_FLIP                    = C.CORACQ_CAP_FLIP
+	CORACQ_PRM_EXT_LINE_TRIGGER_ENABLE = C.CORACQ_PRM_EXT_LINE_TRIGGER_ENABLE
```

Comment:
What do we need this one for?
Is it used for the flip?

Reply:
Aah! late night code changes. No we don't need it.